### PR TITLE
CASSANDRA-16046 - new blog post publicizing community cassandra kubernetes sig

### PR DESCRIPTION
**Pull request by xingh**
content edited / reviewed by different members of the kubernetes sig

### CODE REVIEW

File: src/_posts/2020-08-14-cassandra-and-kubernetes-sig-update.markdown [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+* [Cassandra Operator](https://github.com/sky-uk/cassandra-operator) - A Kubernetes operator by SkyUK that manages Cassandra clusters inside Kubernetes. Well designed and organized. This was among the first operators to be released. 
+* [Instaclustr - Kubernetes Operator for Cassandra operator](https://github.com/instaclustr/cassandra-operator) - The Cassandra operator by Instaclustr manages Cassandra clusters deployed to Kubernetes and automates tasks related to operating an Cassandra cluster.
+* [Cass Operator](https://github.com/datastax/cass-operator) - DataStax's Cassandra Kubernetes Operator supports DataStax as well as open source Apache Cassandra containers on Kubernetes.
```
Comment (jimdickinson) [inline src/_posts/2020-08-14-cassandra-and-kubernetes-sig-update.markdown:24]:
```suggestion
* [Cass Operator](https://github.com/datastax/cass-operator) - DataStax's Kubernetes Operator supports Apache Cassandra as well as DSE containers on Kubernetes. Cassandra configuration is managed directly in the CRD, and Cassandra nodes are managed via a RESTful management API.
```

File: src/_posts/2020-08-14-cassandra-and-kubernetes-sig-update.markdown [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+We’re not starting from zero, as the creators of the five operators are actively participating in the SIG. Hopefully much of the decided upon CRD will have code fragments that can be leveraged from the other projects. The [major operators out publicly today](https://github.com/Anant/awesome-cassandra/blob/master/README.md#deploying-cassandra-on-kubernetes--kubernetized-cassandra) are those by Sky UK, Orange Telecom, Instaclustr, Elassandra, and DataStax (list sourced from the [awesome-cassandra](https://github.com/Anant/awesome-cassandra/blob/master/README.md) project):
+
+* [CassKop - Cassandra Kubernetes Operator](https://github.com/Orange-OpenSource/cassandra-k8s-operator) - This Kubernetes operator by Orange automates Cassandra operations such as deploying a new rack aware cluster, adding/removing nodes, configuring the C and JVM parameters, upgrading JVM and C versions. Written in Go. This one was also one of the first ones out and is the only one that can support multiple clusters using [Multi-CassKop](https://orange-opensource.github.io/casskop/docs/2_setup/3_multi_casskop)
```
Comment (jimdickinson) [inline src/_posts/2020-08-14-cassandra-and-kubernetes-sig-update.markdown:21]:
```suggestion
* [CassKop - Cassandra Kubernetes Operator](https://github.com/Orange-OpenSource/cassandra-k8s-operator) - This Kubernetes operator by Orange automates Cassandra operations such as deploying a new rack aware cluster, adding/removing nodes, configuring the C and JVM parameters, upgrading JVM and C versions. Written in Go. This one was also one of the first ones out and is the only one that can support multiple Kubernetes clusters using [Multi-CassKop](https://orange-opensource.github.io/casskop/docs/2_setup/3_multi_casskop) 
```

**Comment by michaelsembwever** [comment]
@xingh , you’ll need to remove the “merge” commit (we don’t do two-way merging, i.e merges are only to master, rebases off master). and once review has finished the commits will need to be squashed into one…

**Comment by michaelsembwever** [comment]
Merged manually with https://github.com/apache/cassandra-website/commit/ecb519dc1cb84477d1be604a0d88628911807001
 (had to reword the commit message a little)

**Pull request closed without merging**